R/query.R

```diff
@@ -71,6 +74,7 @@ query_parse <- function(expr, context, subquery_env) {
       }
       expr <- expr[[1L]]
     }
+  } else if (is.null(expr)) {
```

Comment:
```suggestion
  } else if (is.null(expr)) {
    # Nothing needed for null expression
```

Reply:
Just wondering if it'll make line 77 look more "intentional" when we look at it later...